### ISSUE TITLE
LG V60 5G: sahara - [LIB]: Unknown error 

### ISSUE THREAD

**Issue by freetink3r** [bkerler/edl#229]
I'm trying to dump an LG V60 5G (LM-V600TM), and edl appears to find the correct loader, but then errors out. Any ideas for how to troubleshoot this?

```
$ ./edl rl dumps --skip=userdata --genxml
Qualcomm Sahara / Firehose Client V3.60 (c) B.Kerler 2018-2022.
main - Trying with no loader given ...
main - Waiting for the device
main - Device detected :)
sahara - Protocol version: 2.1
main - Mode detected: sahara
sahara - 
------------------------
HWID:              0x000c30e100310000 (MSM_ID:0x000c30e1,OEM_ID:0x0031,MODEL_ID:0x0000)
CPU detected:      "SM8250:CD90-PH805-1A"
PK_HASH:           0xe746e34f737403f40212cf29f0c0cab9f1038aa8bce6c097e82cc93213020edb
Serial:            0x1e76093a

sahara - Detected loader: 
/workspace/software/edl/edl/edlclient/../Loaders/LG/000c30e100310000_e746e34f737403f4_fhprg.bin
sahara - Protocol version: 2.1
sahara - Uploading loader 
/workspace/software/edl/edl/edlclient/../Loaders/LG/000c30e100310000_e746e34f737403f4_fhprg.bin ...
sahara - 64-Bit mode detected.
sahara - Firehose mode detected, uploading...
sahara
sahara - [LIB]: Unknown error
Connection detected, quiting.
```

And here's the debug output:

```
$ ./edl --debugmode rl dumps --skip=userdata --genxml
Qualcomm Sahara / Firehose Client V3.60 (c) B.Kerler 2018-2022.
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: list index out of range
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'mprg8626'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'mprg8974'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'mprg8974'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'ramldr'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'ramldr'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'umt'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'prog'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'qualcomm'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'qualcomm'
loader_utils
loader_utils - [LIB]: invalid literal for int() with base 16: 'iqoojtag'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
loader_utils
loader_utils - [LIB]: 'str' object has no attribute 'append'
main - Trying with no loader given ...
main - Waiting for the device
DeviceClass
DeviceClass - [LIB]:   CONFIGURATION 1: 2 mA ====================================
   bLength              :    0x9 (9 bytes)
   bDescriptorType      :    0x2 Configuration
   wTotalLength         :   0x20 (32 bytes)
   bNumInterfaces       :    0x1
   bConfigurationValue  :    0x1
   iConfiguration       :    0x0
   bmAttributes         :   0xa0 Bus Powered, Remote Wakeup
   bMaxPower            :    0x1 (2 mA)
    INTERFACE 0: Vendor Specific ===========================
     bLength            :    0x9 (9 bytes)
     bDescriptorType    :    0x4 Interface
     bInterfaceNumber   :    0x0
     bAlternateSetting  :    0x0
     bNumEndpoints      :    0x2
     bInterfaceClass    :   0xff Vendor Specific
     bInterfaceSubClass :   0xff
     bInterfaceProtocol :   0x10
     iInterface         :    0x0
      ENDPOINT 0x81: Bulk IN ===============================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :   0x81 IN
       bmAttributes     :    0x2 Bulk
       wMaxPacketSize   :  0x200 (512 bytes)
       bInterval        :    0x0
      ENDPOINT 0x1: Bulk OUT ===============================
       bLength          :    0x7 (7 bytes)
       bDescriptorType  :    0x5 Endpoint
       bEndpointAddress :    0x1 OUT
       bmAttributes     :    0x2 Bulk
       wMaxPacketSize   :  0x200 (512 bytes)
       bInterval        :    0x0
DeviceClass
DeviceClass - [LIB]: Detaching kernel driver
main - Device detected :)
DeviceClass
DeviceClass - [LIB]: read:0x30
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 293, in run\n    conninfo = self.doconnect(loop)\n', '  File "./edl", line 207, in doconnect\n    resp = self.sahara.connect()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 111, in connect\n    v = self.cdc.read(length=0xC * 0x4,timeout=1)\n']
DeviceClass
DeviceClass - [LIB]: RX:010000003000000002000000010000000004000000000000000000000000000000000000000000000000000000000000
sahara - Protocol version: 2.1
main - Mode detected: sahara
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 209, in cmd_info\n    if self.enter_command_mode():\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 155, in enter_command_mode\n    if not self.cmd_hello(sahara_mode_t.SAHARA_MODE_COMMAND):\n']
DeviceClass
DeviceClass - [LIB]: TX:020000003000000002000000010000000000000003000000000000000000000000000000000000000000000000000000
DeviceClass
DeviceClass - [LIB]: read:0x200
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 209, in cmd_info\n    if self.enter_command_mode():\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 157, in enter_command_mode\n    res = self.get_rsp()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 64, in get_rsp\n    data = self.cdc.read()\n']
DeviceClass
DeviceClass - [LIB]: RX:0b00000008000000
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 210, in cmd_info\n    self.serial = self.cmdexec_get_serial_num()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 173, in cmdexec_get_serial_num\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_SERIAL_NUM_READ)\n']
DeviceClass
DeviceClass - [LIB]: TX:0d0000000c00000001000000
DeviceClass
DeviceClass - [LIB]: read:0x200
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 210, in cmd_info\n    self.serial = self.cmdexec_get_serial_num()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 173, in cmdexec_get_serial_num\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_SERIAL_NUM_READ)\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 590, in cmd_exec\n    res = self.get_rsp()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 64, in get_rsp\n    data = self.cdc.read()\n']
DeviceClass
DeviceClass - [LIB]: RX:0e000000100000000100000004000000
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 210, in cmd_info\n    self.serial = self.cmdexec_get_serial_num()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 173, in cmdexec_get_serial_num\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_SERIAL_NUM_READ)\n']
DeviceClass
DeviceClass - [LIB]: TX:0f0000000c00000001000000
DeviceClass
DeviceClass - [LIB]: cmd_exec:0x4
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 210, in cmd_info\n    self.serial = self.cmdexec_get_serial_num()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 173, in cmdexec_get_serial_num\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_SERIAL_NUM_READ)\n']
DeviceClass
DeviceClass - [LIB]: RX:3a09761e
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 212, in cmd_info\n    self.hwid = self.cmdexec_get_msm_hwid()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 177, in cmdexec_get_msm_hwid\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_MSM_HW_ID_READ)\n']
DeviceClass
DeviceClass - [LIB]: TX:0d0000000c00000002000000
DeviceClass
DeviceClass - [LIB]: read:0x200
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 212, in cmd_info\n    self.hwid = self.cmdexec_get_msm_hwid()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 177, in cmdexec_get_msm_hwid\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_MSM_HW_ID_READ)\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 590, in cmd_exec\n    res = self.get_rsp()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 64, in get_rsp\n    data = self.cdc.read()\n']
DeviceClass
DeviceClass - [LIB]: RX:0e000000100000000200000018000000
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 212, in cmd_info\n    self.hwid = self.cmdexec_get_msm_hwid()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 177, in cmdexec_get_msm_hwid\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_MSM_HW_ID_READ)\n']
DeviceClass
DeviceClass - [LIB]: TX:0f0000000c00000002000000
DeviceClass
DeviceClass - [LIB]: cmd_exec:0x18
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 212, in cmd_info\n    self.hwid = self.cmdexec_get_msm_hwid()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 177, in cmdexec_get_msm_hwid\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_MSM_HW_ID_READ)\n']
DeviceClass
DeviceClass - [LIB]: RX:00003100e1300c0000003100e1300c0000003100e1300c00
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 213, in cmd_info\n    self.pkhash = self.cmdexec_get_pkhash()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 186, in cmdexec_get_pkhash\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_OEM_PK_HASH_READ)[0:0x20]\n']
DeviceClass
DeviceClass - [LIB]: TX:0d0000000c00000003000000
DeviceClass
DeviceClass - [LIB]: read:0x200
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 213, in cmd_info\n    self.pkhash = self.cmdexec_get_pkhash()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 186, in cmdexec_get_pkhash\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_OEM_PK_HASH_READ)[0:0x20]\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 590, in cmd_exec\n    res = self.get_rsp()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 64, in get_rsp\n    data = self.cdc.read()\n']
DeviceClass
DeviceClass - [LIB]: RX:0e000000100000000300000090000000
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 213, in cmd_info\n    self.pkhash = self.cmdexec_get_pkhash()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 186, in cmdexec_get_pkhash\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_OEM_PK_HASH_READ)[0:0x20]\n']
DeviceClass
DeviceClass - [LIB]: TX:0f0000000c00000003000000
DeviceClass
DeviceClass - [LIB]: cmd_exec:0x90
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 213, in cmd_info\n    self.pkhash = self.cmdexec_get_pkhash()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 186, in cmdexec_get_pkhash\n    res = self.cmd_exec(exec_cmd_t.SAHARA_EXEC_CMD_OEM_PK_HASH_READ)[0:0x20]\n']
DeviceClass
DeviceClass - [LIB]: RX:e746e34f737403f40212cf29f0c0cab9f1038aa8bce6c097e82cc93213020edbcb967bfe180ae14c9afb4dac9c83e2b0e746e34f737403f40212cf29f0c0cab9f1038aa8bce6c097e82cc93213020edbcb967bfe180ae14c9afb4dac9c83e2b0e746e34f737403f40212cf29f0c0cab9f1038aa8bce6c097e82cc93213020edbcb967bfe180ae14c9afb4dac9c83e2b0
sahara -
------------------------
HWID:              0x000c30e100310000 (MSM_ID:0x000c30e1,OEM_ID:0x0031,MODEL_ID:0x0000)
CPU detected:      "SM8250:CD90-PH805-1A"
PK_HASH:           0xe746e34f737403f40212cf29f0c0cab9f1038aa8bce6c097e82cc93213020edb
Serial:            0x1e76093a

sahara - Detected loader: /workspace/software/edl/edl/edlclient/../Loaders/LG/000c30e100310000_e746e34f737403f4_fhprg.bin
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 328, in run\n    sahara_info = self.sahara.cmd_info()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 304, in cmd_info\n    self.cmd_modeswitch(sahara_mode_t.SAHARA_MODE_COMMAND)\n']
DeviceClass
DeviceClass - [LIB]: TX:0c0000000c00000003000000
DeviceClass
DeviceClass - [LIB]: read:0x30
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 330, in run\n    resp = self.sahara.connect()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 111, in connect\n    v = self.cdc.read(length=0xC * 0x4,timeout=1)\n']
DeviceClass
DeviceClass - [LIB]: RX:010000003000000002000000010000000004000003000000000000000000000000000000000000000000000000000000
sahara - Protocol version: 2.1
sahara - Uploading loader /workspace/software/edl/edl/edlclient/../Loaders/LG/000c30e100310000_e746e34f737403f4_fhprg.bin ...
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 335, in run\n    mode = self.sahara.upload_loader()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 508, in upload_loader\n    if not self.cmd_hello(sahara_mode_t.SAHARA_MODE_IMAGE_TX_PENDING):\n']
DeviceClass
DeviceClass - [LIB]: TX:020000003000000002000000010000000000000000000000000000000000000000000000000000000000000000000000
DeviceClass
DeviceClass - [LIB]: read:0x200
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 335, in run\n    mode = self.sahara.upload_loader()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 516, in upload_loader\n    resp = self.get_rsp()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 64, in get_rsp\n    data = self.cdc.read()\n']
DeviceClass
DeviceClass - [LIB]: RX:12000000200000000d0000000000000000000000000000004000000000000000
sahara - 64-Bit mode detected.
sahara - Firehose mode detected, uploading...
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 335, in run\n    mode = self.sahara.upload_loader()\n']
DeviceClass
DeviceClass - [LIB]: TX:7f454c4601010100000000000000000002002800010000000000a09f340000000000000000000000340020000300000000000000000000000000000000000000
DeviceClass
DeviceClass - [LIB]: read:0x200
DeviceClass
DeviceClass - [LIB]: ['  File "./edl", line 380, in <module>\n    base.run()\n', '  File "./edl", line 335, in run\n    mode = self.sahara.upload_loader()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 516, in upload_loader\n    resp = self.get_rsp()\n', '  File "/workspace/software/edl/edl/edlclient/Library/sahara.py", line 64, in get_rsp\n    data = self.cdc.read()\n']
DeviceClass
DeviceClass - [LIB]: RX:04000000100000000d00000029000000
sahara
sahara - [LIB]: Unknown error
Connection detected, quiting.
```

**Comment by Baddad23** [comment]
I’ll just give you a heads up since support here isn’t what you”d hope for. The loader  here ain’t for shit, believe me I know. Your best bet is to head over to xda , look up the lg v60 forum. There you find the firehose file for the v60. It’s an .elf file.. just use lt. I know It works, I have 2 v60.

**Comment by GustavoCodesAgain** [comment]
> I’ll just give you a heads up since support here isn’t what you”d hope for. The loader here ain’t for shit, believe me I know. Your best bet is to head over to xda , look up the lg v60 forum. There you find the firehose file for the v60. It’s an .elf file.. just use lt. I know It works, I have 2 v60.

How did you made id it work? im stuck in Mode detected: Sahara and doesn't continue

**Comment by freetink3r** [comment]
@Baddad23 wanted to follow up and thanks for the pointer. I found the LG V60 loader [here](https://drive.google.com/file/d/1VqSU6bezKu0h5OiVaTaZc88ctLOt83_e/view?usp=sharing) and was able to connect with the edl tool. One wrinkle was that I wasn't able to boot into EDL mode until I clicked "Enable OEM unlock" in Developer Options. I don't think all phones require that to boot into EDL mode so thought I would mention it for the next guy.
